modules/snooker/src/main/ts/ephox/snooker/queries/CellOperations.ts

```diff
@@ -0,0 +1,56 @@
+import { TableSelection } from '@ephox/darwin';
```

Comment:
Just realized this won't work for two reasons: darwin isn't a dependency on snooker (see package.json) and adding that would create a circular dependency whereby someone likely wouldn't be able to install.

However, thankfully `TableSelection` is just an alias to `SelectorFilter.descendants(container, selector)` and then wrapped in an `Option`. So we can just us that logic directly here instead to work around that.

Reply:
Actually, on that note... given Darwin is meant to be used for selection, maybe this code should be there, instead of in Snooker?

Reply:
Just copying this across from slack:
> as a heads up, the Ephemera type already exists in Darwin
> that also makes me believe even more that table selection, etc... code should be in darwin instead of snooker
> https://github.com/tinymce/tinymce/blob/develop/modules/darwin/src/main/ts/ephox/darwin/api/Ephemera.ts#L3 for reference

Reply:
I'm really starting to agree that this should be in darwin... Will look into doing that.